scipy/stats/stats.py

```diff
@@ -1394,6 +1394,20 @@ def normaltest(a, axis=0, nan_policy='propagate'):
     .. [2] D'Agostino, R. and Pearson, E. S. (1973), "Tests for departure from
            normality", Biometrika, 60, 613-622
 
+    Examples
+    --------
+    >>> import scipy.stats as stats
```

Comment:
`from scipy import stats` is simpler. (["Don't repeat yourself"](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself); why write `stats` twice?)